facebook/docusaurus
# docs(v2): Add CountryStateCityAPI site to showcase page

## Motivation

Makes the site up and running within 30 mins.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

* Run `yarn start:website`
* Check that the page http://localhost:3000/showcase is rendered properly.

## Related PRs

None
